### PR TITLE
Continuous integration

### PR DESCRIPTION
**Pull request by valtri**
Implemented build and publish using chartpress.

The https://egi-federation.github.io/egi-notebooks-accounting would be a helm repository. But there has been used the https://egi-federation.github.io/egi-notebooks-chart/ before - it would require small change here yet (removing the chartpress modification hack, and adding a new secret).

Additional steps:
* creating gh-pages branch, or decide to use the shared egi-notebooks-chart
* creating secrets `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN`
* probably a new tag, something like "1.0.0"? (major version bump after the "0.1.1")

### CODE REVIEW

File: .github/workflows/flake8.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: Python Flake8 Checks
```
Comment (enolfc) [inline .github/workflows/flake8.yml]: why not just use the superlinter as with other repos? it will complain about more things :)

Comment (valtri) [reply .github/workflows/flake8.yml]: Switched to super-linter. It's true it complains a alot. :-) Combinantion of templating with yaml/k8s seems problematic, but many linting issues was resolved.

**Review by enolfc — APPROVED** [review]
LGTM

**Comment by enolfc** [comment]
I assume this is ok to merge, did a quick review.

**Pull request merged**